stackslib/src/net/neighbors/comms.rs

```diff
@@ -466,12 +466,19 @@ impl PeerNetworkComms {
                 Ok(None) => {
                     if let Some(rh) = req_opt {
                         // keep trying
+                        debug!("{:?}: keep polling {}", network.get_local_peer(), naddr);
```

Comment:
Kind of unrelated to this PR, but given that the `p2p` and `relay` threads include `(p2p_bind, rpc_bind)` and `data_url` respectively, is logging `network.get_local_peer()` all that helpful?

Reply:
The tests don't, and the tests often have multiple `TestPeer`s running

Reply:
Okay -- that's fine for this PR then. I'll open an issue, because those tests could probably do the same thing with  thread names, and it would make the networking logs much easier to read through.